.github/workflows/nightwatch.yml

```diff
@@ -5,22 +5,22 @@ on:
       e2e:
```

Comment:
@tim-yao Do we need this `e2e` input and all of the `if: inputs.e2e` conditions on each e2e job anymore?

Reply:
FYI Estuary does not send this parameter input to the GitHub actions so with the current logic the `e2e` jobs will not execute.

Reply:
> @tim-yao Do we need this `e2e` input and all of the `if: inputs.e2e` conditions on each e2e job anymore?
no, because the GHA API calls the specific workflow directly, so we don't need this parameter anymore.